ext/pdo_sqlite/pdo_sqlite_class.c

```diff
@@ -0,0 +1,657 @@
+
+/*
+  +----------------------------------------------------------------------+
+  | Copyright (c) The PHP Group                                          |
+  +----------------------------------------------------------------------+
+  | This source file is subject to version 3.01 of the PHP license,      |
+  | that is bundled with this package in the file LICENSE, and is        |
+  | available through the world-wide-web at the following url:           |
+  | https://www.php.net/license/3_01.txt                                 |
+  | If you did not receive a copy of the PHP license and are unable to   |
+  | obtain it through the world-wide-web, please send a note to          |
+  | license@php.net so we can mail you a copy immediately.               |
+  +----------------------------------------------------------------------+
+  | Author: Danack
+  +----------------------------------------------------------------------+
+*/
+
+/* The PDO Database Handle Class */
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "php.h"
+#include "../pdo/php_pdo.h"
+#include "../pdo/php_pdo_driver.h"
+#include "../pdo/php_pdo_int.h"
+#include "zend_exceptions.h"
+#include "zend_object_handlers.h"
+#include "zend_hash.h"
+
+#include "../pdo_sqlite/php_pdo_sqlite.h"
+#include "../pdo_sqlite/php_pdo_sqlite_int.h"
+
+typedef struct {
+	zval val;
+	zend_long row;
+} pdopgsql_aggregate_context;
+
+static int do_callback(struct pdo_sqlite_fci *fc, zval *cb,
+		int argc, sqlite3_value **argv, sqlite3_context *context,
```

Comment:
Standard type we use for ``argc`` is ``uint32_t`` this means this should also probably be the type for ``i`` and ``fake_argc``

```suggestion
		uint32_t argc, sqlite3_value **argv, sqlite3_context *context,
```

Reply:
That's fair